src/main/java/com/jmb/springfactory/model/entity/WorkLog.java

```diff
@@ -18,10 +15,15 @@
 
   private Double hoursWorked;
 
-  @OneToOne(fetch = FetchType.LAZY)
-  private Comment comment;
+  private String description;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "description" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)